iina/PlayerCore.swift

```diff
@@ -443,6 +443,16 @@ class PlayerCore: NSObject {
     sendOSD(.abLoop(info.abLoopStatus))
   }
 
+  func clearAbLoop() {
+    if let resetOption = mpv.getString(MPVOption.ProgramBehavior.resetOnNextFile) {
+        if resetOption.contains(MPVOption.PlaybackControl.abLoopA) &&
+                   resetOption.contains(MPVOption.PlaybackControl.abLoopB) {
+          info.abLoopStatus = 0
+          sendOSD(.abLoop(info.abLoopStatus))
```

Comment:
Maybe I didn't make it clear. You don't need to send notification unless you listen to mpv's `ab-loop` change event and update IINA's state at runtime.

Reply:
I took a look at the callback handle. in `MPVController::handleEvent`, seems when I set AB loop there is no event caught by readEvent